proto_patch.sed

```diff
@@ -24,15 +24,8 @@ s+repeated opentelemetry.proto.common.v1.KeyValue \(.*\);+repeated opentelemetry
 s+repeated KeyValue \(.*\);+repeated KeyValue \1\
   [ (gogoproto.nullable) = false ];+g
 
-s+repeated opentelemetry.proto.common.v1.AnyValue \(.*\);+repeated opentelemetry.proto.common.v1.AnyValue \1\
-  [\
-  (gogoproto.nullable) = false\
-  ];+g
-
 s+AnyValue \(.*\);+AnyValue \1\
-  [\
-  (gogoproto.nullable) = false\
-  ];+g
```

Comment:
Did you run `make genproto` after this change?

Reply:
Yes